internal/imagedefinition/image_definition.go

```diff
@@ -90,15 +84,9 @@ type Installer struct {
 
 // CloudInit provides customizations for running cloud-init
 type CloudInit struct {
-	MetaData      string      `yaml:"meta-data"      json:"MetaData,omitempty"`
-	UserData      *[]UserData `yaml:"user-data"      json:"UserData,omitempty"`
-	NetworkConfig string      `yaml:"network-config" json:"NetworkConfig,omitempty"`
-}
-
-// UserData defines the user information to be used by cloud-init
-type UserData struct {
-	UserName     string `yaml:"name"     json:"UserName"`
-	UserPassword string `yaml:"password" json:"UserPassword"`
+	MetaData      string `yaml:"meta-data"      json:"MetaData,omitempty"`
+	UserData      string `yaml:"user-data"      json:"UserData,omitempty"`
+	NetworkConfig string `yaml:"network-config" json:"NetworkConfig,omitempty"`
 }
```

Comment:
Should we include `vendor-data` here too? Technically it *is* a part of the cloud-init seed. We've never used it in our images, and it is optional, but it's part of the seed and so presumably someone, somewhere, might want to use it?